packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -63,7 +74,38 @@ class Chat extends Widget<Properties> {
   _renderMessageBox(): void {
     const $messageBox = $('<div>').appendTo(this.element());
 
-    this._messageBox = this._createComponent($messageBox, MessageBox, {});
+    const configuration: MessageBoxProperties = {
+      onMessageSend: (text) => {
+        this._messageSendHandler(text);
+      },
+    };
+
+    this._messageBox = this._createComponent($messageBox, MessageBox, configuration);
+  }
+
+  _createMessageSendAction(): void {
+    this._messageSendAction = this._createActionByOption(
+      'onMessageSend',
+      { excludeValidators: ['disabled', 'readOnly'] },
+    );
+  }
+
+  // eslint-disable-next-line max-len
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
+  _messageSendHandler(action: any): void {
+    const { text, event } = action;
+    // @ts-expect-error
+    const { user } = this.option();
```

Comment:
Why didn't we add user option in our types yet?

Reply:
We can't do it without a Technical Writer. He's not available right now. The PR was set up for this a long time ago